src/app/about/page.js

```diff
@@ -2,7 +2,7 @@ import AboutUs from "@/components/AboutUs/AboutUs";
 
 const AboutUss = () => {
```

Comment:
Please rename the page name give a meaningful name of the page, Like AboutUsPage